src/navx_node.cpp

```diff
@@ -21,6 +21,12 @@
 #include "ahrs/AHRS.h"
 
 #define DEG_TO_RAD ((2.0 * 3.14159) / 360.0)
+#define GRAVITY 9.81 // m/s^2, if we actually go to the moon remember to change this
+typedef struct {
+  float ypr[3];
+  float ang_vel[3];
+  float accel[3];
+} Pose;
```

Comment:
It would be better to name it something other than the generic "Pose": for example, there are already geometry_msgs msgs with that name as well.